src/components/ens-registration/IntroMarquee/IntroMarquee.tsx

```diff
@@ -79,13 +79,17 @@ export default function IntroMarquee({
 }
 
 function ENSAvatarPlaceholder({
-  name,
+  account,
   onPress,
   onPressCancel,
   onPressStart,
   testID,
 }: {
-  name: string;
+  account: {
+    name: string;
+    address: EthereumAddress;
+    avatar: string;
+  };
```

Comment:
Can we refer to a generated type value here instead of duplicating the definition?